develop/dev-guide-bi-looker-stduio.md

```diff
@@ -0,0 +1,128 @@
+
+    ![Looker Studio: configure connection settings for TiDB Serverless](/media/develop/looker-studio-configure-connection.png)
+
+7. Click **AUTHENTICATE**. 
+
+If the authentication succeeds, you can see tables in the database. 
+
+## Create a simple chart
+
+Now, you can use TiDB Serverless as a data source. Let's make a simple chart with data. 
+
+1. In the setting pane, select **CUSTOM QUERY**. 
+
+2. Put the SQL below, then click **Add** in the bottom-right corner. 
+    ```sql
+    SELECT sector,
+        COUNT(*)                                                                      AS companies,
+        ROW_NUMBER() OVER (ORDER BY COUNT(*) DESC )                                   AS companies_ranking,
+        SUM(market_cap)                                                               AS total_market_cap,
+        ROW_NUMBER() OVER (ORDER BY SUM(market_cap) DESC )                            AS total_market_cap_ranking,
+        SUM(revenue_growth * weight) / SUM(weight)                                    AS avg_revenue_growth,
+        ROW_NUMBER() OVER (ORDER BY SUM(revenue_growth * weight) / SUM(weight) DESC ) AS avg_revenue_growth_ranking
+    FROM companies
+        LEFT JOIN index_compositions ic ON companies.stock_symbol = ic.stock_symbol
+    GROUP BY sector
+    ORDER BY 5 ASC
+    ```
+
+    If you see **You are about to add data to this report** dialog, click **ADD TO REPORT**.
+
+3. You see a table in the report. Click **Add a chart**, then select `Combo chart` in the `Line` category.
+
+4. In the **Chart** settings pane, configure the following parameters:
+
+    - **Dimension**: `sector`.
+    - **Metric**: `companies` and `total_market_cap`.
+    - In the **STYLE** Tab:
+      - Series #1 check `Line`.
+      - Series #1 axis `Right`.
+      - Series #2 check `Bars`.
+      - Series #2 axis `Left`.
+    - Leave others as defaults.
+
+You can see a combo chart below:
+
+![Looker Studio: A simple Combo Chart](/media/develop/looker-studio-simple-chart.png)
+
+## Next steps
+
+- Learn more usage of Looker Studio from [Looker Studio Help](https://support.google.com/looker-studio)
+- Learn the best practices for TiDB application development with the chapters in the [Developer guide](/develop/dev-guide-overview.md), such as [Insert data](/develop/dev-guide-insert-data.md), [Update data](/develop/dev-guide-update-data.md), [Delete data](/develop/dev-guide-delete-data.md), [Single table reading](/develop/dev-guide-get-data-from-single-table.md), [Transactions](/develop/dev-guide-transaction-overview.md), and [SQL performance optimization](/develop/dev-guide-optimize-sql-overview.md).
+- Learn through the professional [TiDB developer courses](https://www.pingcap.com/education/) and earn [TiDB certifications](https://www.pingcap.com/education/certification/) after passing the exam.
+
+## Need help?
+
+Ask questions on [Discord](https://discord.gg/DQZ2dy3cuc?utm_source=doc) or [create a support ticket](https://support.pingcap.com/).
```

Comment:
```suggestion
Ask questions on [Discord](https://discord.gg/DQZ2dy3cuc?utm_source=doc) or [create a support ticket](/support.md).
```

Reply:
This can't be resolved due to outdated, but it was fixed.

Reply:
@dveeden This change broke ci/tidb-cloud-check. reverted.